packages/frontend/src/pages/Settings.tsx

```diff
@@ -69,10 +68,6 @@ const Settings: FC = () => {
         FeatureFlags.PassthroughLogin,
     );
 
-    const isSemanticLayerEnabled = useFeatureFlagEnabled(
-        FeatureFlags.SemanticLayerEnabled,
```

Comment:
We can also remove the feature flag from packages/common/src/types/featureFlags.ts